### PR TITLE
Add client status for persons

### PR DESCRIPTION
**Pull request by ibnu2651**

### CODE REVIEW

**Comment by ibnu2651** [comment]
Fixes #59.

File: src/main/java/seedu/address/model/person/ClientStatus.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        return value.ordinal();
+    }
+
+    public ClientStatus increment() {
+        return new ClientStatus(Math.min(this.getStatus() + 1, CLIENT_MAX));
+    }
+
+    public ClientStatus decrement() {
+        return new ClientStatus(Math.max(this.getStatus() - 1, CLIENT_MIN));
+    }
```
Comment (kaiyi27) [inline src/main/java/seedu/address/model/person/ClientStatus.java:59]: Maybe can consider adding javadoc to public method and constructor?

File: src/test/java/seedu/address/logic/commands/ClientStatusCommandTest.java
Comment (kaiyi27) [inline src/test/java/seedu/address/logic/commands/ClientStatusCommandTest.java:1]: Good testing on the method in ClientStatusCommand. Handle different situation and tested well.

**Review by kaiyi27 — COMMENTED** [review]
Overall LGTM. The code is neat and clean. The code is also well-structured and follows good coding practices, with clear comments explaining the functionality of each section. Error handling is effectively implemented.

**Pull request merged**